packages/suite/src/hooks/wallet/__tests__/useSendForm.test.tsx

```diff
@@ -357,7 +357,9 @@ describe('useSendForm hook', () => {
     });
 
     fixtures.feeChange.forEach(f => {
-        it(
```

Comment:
I looked at this tests like on Friday and I even tried to rewrite it with `renderHook`. But I was like: nah, I can't even tell what does it test.

So I skipped that, but I would really advise to rewrite it so that we only test the hook itself.